src/hyperfine/timer/cputimer.rs

```diff
@@ -45,9 +45,53 @@ fn get_cpu_times() -> CPUTimes {
 /// Read CPU execution times (dummy for now)
 #[cfg(windows)]
 fn get_cpu_times() -> CPUTimes {
+    use winapi::um::processthreadsapi::{GetCurrentProcess, GetProcessTimes};
+    use winapi::shared::minwindef::FILETIME;
+
+    // Winapi reports times as per 100 nanosecond
+    const HUNDRED_NS_PER_MS: i64 = 10;
+    
+    let mut _ctime = FILETIME {
+        dwLowDateTime: 0,
+        dwHighDateTime: 0,
+    };
+    let mut _etime = FILETIME {
+        dwLowDateTime: 0,
+        dwHighDateTime: 0,
+    };
+    let mut kernel_time = FILETIME {
+        dwLowDateTime: 0,
+        dwHighDateTime: 0,
+    };
+    let mut user_time = FILETIME {
+        dwLowDateTime: 0,
+        dwHighDateTime: 0,
+    };
+
+    let (user_usec, system_usec) = unsafe {
+        let handle = GetCurrentProcess();
```

Comment:
I would have somehow expected that we need to get the PID of the child process here (which would probably require some restructuring).

Are you sure that this will measure CPU times of the child process (or that this is equivalent)?

Reply:
This is a good question, that I honestly don't know 100% for sure about. My initial hope is that as each command we spawn is treated as a child process of our current handle and that we window the requests, we can piggyback and obtain the desired information. However, this may not actually be the case.

Otherwise we may have to use [CreateProcess](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682425(v=vs.85).aspx) from the winapi crate and possible use the procedure outlined [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682499(v=vs.85).aspx) to obtain the proper information with a handle.

Reply:
However, it does appear that when a child process is created the parent handle is [inherited](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684868(v=vs.85).aspx). Whether this means `GetProcessInformation` is giving timing information back as expected I'm not sure.